### PR TITLE
Update entrypoint.py

### PR DESCRIPTION
**Pull request by DeepeshHomex**

### CODE REVIEW

File: .github/actions/process-review-comments/entrypoint.py [hunk 1]
```diff
@@ -29,7 +29,9 @@ def main():
     resp = requests.get(url=url, headers=headers)
     data = resp.json()
     print(data)
-
+    
+    for comment in data:
```
Comment (deepeshp656) [inline .github/actions/process-review-comments/entrypoint.py:55]: lets check this ok

File: .github/actions/process-review-comments/entrypoint.py [hunk 2]
```diff
@@ -29,7 +29,9 @@ def main():
...
+        print(data['path'])
```
Comment (deepeshp656) [inline .github/actions/process-review-comments/entrypoint.py]: adding reply

File: .github/actions/process-review-comments/entrypoint.py (hunk 1 above)
Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:55]: see

File: .github/actions/process-review-comments/entrypoint.py (hunk 2 above)
Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py]: lhlhk

File: .github/actions/process-review-comments/entrypoint.py [hunk 3]
```diff
@@ -12,10 +12,19 @@ def setup_args():
 
     return parser.parse_args()
 
+def review_comment_check(comment_body):
+    print(comment_body)
+    res = comment_body.startswith(tuple(pref_list))
```
Comment (deepeshp656) [inline .github/actions/process-review-comments/entrypoint.py:18]: Change

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:18]: knk,

File: .github/actions/process-review-comments/entrypoint.py [hunk 4]
```diff
@@ -15,6 +15,7 @@ def setup_args():
 def review_comment_check(comment_body):
     print(comment_body)
     res = comment_body.startswith(tuple(pref_list)) 
+    print(res)
```
Comment (deepeshp656) [inline .github/actions/process-review-comments/entrypoint.py:19]: Question

File: .github/actions/process-review-comments/entrypoint.py (hunk 1 above)
Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:55]: ????

File: .github/actions/process-review-comments/entrypoint.py (hunk 2 above)
Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py]: testing

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py]: ihlk

File: .github/actions/process-review-comments/entrypoint.py (hunk 3 above)
Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:18]: lljl;

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:18]: seee

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:18]: mjm

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:18]: next

Comment (DeepeshHomex) [reply .github/actions/process-review-comments/entrypoint.py:18]: khlk

Comment (DeepeshHomex) [reply .github/actions/process-review-comments/entrypoint.py:18]: ???

File: .github/actions/process-review-comments/entrypoint.py (hunk 4 above)
Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:19]: lkhlohlhl;

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:19]: nbmn

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:19]: hkkhkklnlkjl

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:19]: yeyy

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:19]: jlkj

File: .github/actions/process-review-comments/entrypoint.py [hunk 5]
```diff
@@ -2,6 +2,7 @@
 import argparse
 import os
 import requests
+import json
```
Comment (deepeshp656) [inline .github/actions/process-review-comments/entrypoint.py:5]: Test

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:5]: jhjhkklml

File: .github/actions/process-review-comments/entrypoint.py (hunk 3 above)
Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:18]: test

File: .github/actions/process-review-comments/entrypoint.py (hunk 4 above)
Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:19]: .

**Pull request merged**